include/pokeagb/pokemon/ability.h

```diff
@@ -90,8 +90,163 @@ enum PokemonAbility {
     ABILITY_WHITE_SMOKE,
     ABILITY_PURE_POWER,
     ABILITY_SHELL_ARMOR,
-    ABILITY_CACOPHONY,
     ABILITY_AIR_LOCK,
+    ABILITY_TANGLED_FEET,
```

Comment:
Please remove - anything that's not in a base generation 3 ROM should not be added and should be overridden in your own code